src/stores/Widgets/SpectralLineQueryWidgetStore/SpectralLineQueryWidgetStore.ts

```diff
@@ -317,7 +317,12 @@ export class SpectralLineQueryWidgetStore {
             if (filterString !== "") {
                 const column = this.queryResult.get(controlHeader.columnIndex);
```

Comment:
How about just replacing ```queryResult``` with ```filterResult```? My quick test works, but I am not sure if there are other effects.

Reply:
I saw the effect. Forget about this.